.gitmodules

```diff
@@ -11,3 +11,6 @@
 [submodule "plugins/obs-browser"]
 	path = plugins/obs-browser
 	url = https://github.com/kc5nra/obs-browser.git
+[submodule "plugins/obs-vst"]
+	path = plugins/obs-vst
+	url = git@github.com:DDRBoxman/obs-vst.git
```

Comment:
You should probably replace this with an http URL like all the other submodules.